rskj-core/src/main/java/co/rsk/peg/federation/FederationTracker.java

```diff
@@ -0,0 +1,27 @@
+package co.rsk.peg.federation;
+
+public class FederationTracker<T> {
```

Comment:
We could have `T extends Federation` just to make this more explicit. We will need to have PendingFederation extend Federation. Maybe we can look more into this.